src/OpenTelemetry.Instrumentation.AspNet/CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## Unreleased
 
+* Leverage AddLegacySource api from OpenTelemetrySDK instead of
```

Comment:
What does this mean for the reader? (what do we expect the reader to get)?

Reply:
For most users who just used the instrumentation without any customization - probably nothing.
But for advanced users who updates to this version (from prior) will see differences - eg: Sampling/kicking off processor pipeline is now done by the TracerProvider itself, before instrumentation gets the activity. So if one wrote a custom sampler or custom ActivityProcessor.OnStart, that would see activity earlier, even before its Kind/Source/DisplayName is updated.

Reply:
This seems to be a better description that we want to add to changelog? "Sampling/kicking off processor pipeline is now done by the TracerProvider itself, before instrumentation gets the activity."

Reply:
I'll adjust this.

Reply:
Reworded.
@utpilla we need to update the PR description in these PRs to reflect user impact - like the Samplers/Processor.OnStart being called *before* instrumentation.